### PR TITLE
6958 talk info in talks

### PR DESCRIPTION
**Pull request by TanyaGordiychuk**

### CODE REVIEW

File: conference/react-js/src/components/EventCard/EventCard.jsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+import PropTypes from 'prop-types';
+import { myEvents } from '../../constants/route-url';
+
+class EventCard extends PureComponent {
+  setConferenceData = () => {
+    this.props.setConference(this.props.data);
+  };
+
+  render() {
+    const { call_for_paper_end_date: callForPaperEndDate,
```
Comment (SupraVlad) [inline conference/react-js/src/components/EventCard/EventCard.jsx:12]: we use camelCase in naming variables and functions

File: conference/react-js/src/components/Talks/ReviewTalk/ReviewTalk.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  <div className="pop-up-wrapper">
+    <div className="pop-up review-talk">
+      <div className="review-talk__topic">
+        JVM Languages and new programming paradigms
+      </div>
+      <div className="review-talk__type">
+        Regular Talk:
+      </div>
+      <div className="review-talk__title">
+        Why Java Sucks
```
Comment (SupraVlad) [inline conference/react-js/src/components/Talks/ReviewTalk/ReviewTalk.jsx]: remove, please.

File: conference/react-js/src/components/Talks/ReviewTalk/ReviewTalk.jsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      </div>
+      <p className="review-talk__description">
+        Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore
```
Comment (SupraVlad) [inline conference/react-js/src/components/Talks/ReviewTalk/ReviewTalk.jsx]: is it necessary to use mock data in implementation? I think it should be removed.

File: conference/react-js/src/components/Talks/ReviewTalk/review-talk.scss [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+			background: $raspberry;
+
+			&:hover {
+				background: darken($raspberry, 20%);
+			}
+		}
+	}
+}
+
+//.additional-info {
```
Comment (SupraVlad) [inline conference/react-js/src/components/Talks/ReviewTalk/review-talk.scss]: remove commented code

File: conference/react-js/src/containers/EventTalks/EventTalks.jsx [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    </Link>
+    <Talks
+      columns={columnsList}
+      onClick={onClick}
+      url={`/conference/${id}/talks`}
+    />
+  </div>
+);
+
+EventTalks.propTypes = { conference: PropTypes.shape({
```
Comment (SupraVlad) [inline conference/react-js/src/containers/EventTalks/EventTalks.jsx]: provide please tabulation. move `conference` to the new line. it is hard to read this block

**Review by SupraVlad — CHANGES_REQUESTED** [review]

File: conference/react-js/src/components/Talks/ReviewTalk/ReviewTalk.jsx (hunk 3 above)
Comment (TanyaGordiychuk) [reply conference/react-js/src/components/Talks/ReviewTalk/ReviewTalk.jsx]: I need this data just to review pop-up design

**Review by kris45 — APPROVED** [review]

**Review by SupraVlad — APPROVED** [review]

**Pull request merged**